admin-dev/themes/default/scss/partials/_nav.scss

```diff
@@ -16,64 +14,78 @@
+        cursor: pointer;
       }
 
       &__header {
         display: flex;
+        gap: var(--#{$cdk}size-16);
         align-items: center;
       }
 
       .logo {
-        width: 8rem;
-        height: 1.25rem;
-        background: url("#{$img-path}/ps-logo-white.svg") no-repeat;
+        width: var(--#{$cdk}size-128);
+        height: var(--#{$cdk}size-24);
+        //background: url("#{$img-path}/ps-logo-white.svg") no-repeat;
+        background: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAUYAAAAkCAMAAADCQmMnAAAAAXNSR0IB2cksfwAAAAlwSFlzAAAWJQAAFiUBSVIk8AAAAEVQTFRFHR0bHR0bHR0bHR0bHR0bHR0bAAAAHR0bHR0bHR0bHR0bHR0bHR0bHR0bHR0bHR0bHR0bHR0bHR0bHR0bHR0bHR0bHR0buFvkHQAAABd0Uk5Tv6+fgFAgAEB/3+//z2AQj3BvXzCQoLC2y4BfAAAGb0lEQVR4nLWa6aKrKAyA3TngUu/oue//qFO1ZCNBOu3kXwGzfOyhVcWlbtquH4g0N/XDUBnS0Ubup/ZhHMNUz+RzZ317WIJWP1jYDJos7jH5cRx99egWVvOIH7Z3hZc/6+mlouhpJCXloE01KlLN+LVX6v1MbWgaDkEly09g6l2smK1vDyPwOf1YBpdqb2gvQ3jNXeGhh8fa8OHSa07WLoPxGQOo0DAeIN/A6BId0cMijKzRmlD8E+SXP/8JY6qHg1QxxjYGxnH8J4txHB/FGH+UOr+VY6xpaSUgLo3y6QThF2NcVBCeLEwGxisUE2PkaGFEH6wGs03xuUhupRiF+2JW1/q3sVUpxt4KE0e2hfEMxcYY+jxGmF9W/YXRGbVnpCUYRRu+Leh99Fyz3sNoUiQcTYxHKDbG1/yxDbw432A0v38UYhQesllthyaX/izGxQ4SNGVsPbsWnfSHsGXWMQzhbEDr9wKMHfwMbbe5mUBZijAm3jsCBBdGv7rNrehe9Q7GB7P7PMzQ32GRjiSkAllZL/f6Fet3hvG11M2efM0x7o7JwgKNe38LzZ+LQg+NfaLk2oSGVVBksxq+qi9Xll9o1pdjpD21LzLKtNWlut+xySox0sXMaxiHZRKfjLIFFWgMJxWw+Je2S8wkFVEC1mFkW3QucD1FGJFBgKG+kL2rVzHSHv6bYhywR/X4kPN6jzEklTDNpxKMWyxXHCWewAkHhkhbjBFMYG8cgqNlNzCiIq94N7P2SnyApr3HCJUQxzK/hF0WLYywKHTg6a5ghLG+Re1bMUZcYNkpoIcwr8VLwYjTWsHI+1iJD4qae4w4JxunVN9ijOUBF1Wc1WRREzc3g5haCKY9/xZXcceNgSkccArGjrX/cDTSTS/Uq1PuxDmM0KUNmXu4gBHlo2/mLVEL5usehWNEvSLvgdx2AyOCVjDiINfjM9fGSUnvoKXYqJm1gWNgxDmdTAJGCcIRmZnMsTgqwlEjpwsYrAyMoH1KMWLzSY/P3KmpxNbqJWZaE5IGRiimPRKAVNJJZ9BERQFG1CFnCuy1XsdI5rTEuMx4bmy0+Bw5sA/3GI1L7/gQPusYYaT8ZT0CbYzrB+afCjDCPiGWRgpYw7iQiyi5VZxTkh3O+S0mrW9KMFoXLc8HpI6xYaV8kp2ypdmtU2L+qQAjH3NU8FzIMZ6rFgurz9iZeHyJZO/USMO69ns2HlWMuIUsfHTgp5uh/fE/YlT0ZOx8lOGhNGZdBzulqRhh8bkuPMox8Rnd76iK+xyjNamlPCeWbacV8UmBU7Bay3eKuVYmX6DDUcVYiUJlVh/xzVoQVSlGe4uBZV1sMVJcxk7scAsj3iXQ8wYlOWsfr0VCBSWmYUTH2+tmgs6KiJdur2RHLQxjIIcxaHlixP1W+gxHEnHg4RJmu7s8qMyvPQyjdvxm0ne/VN1OqjSMmTRa8qb3lI131MYw2reY3lJqHr+ZVPZbzLRibxsYybSCsluMZ7BokYamYZxGU+STTBSyDM+FGAdxdwbBa0hnYqySl8EpPr/yszHeOPk2jzMgh7G6cpxTRSYMn1M2xszW+DI/Ru2oB692xRgxwcBTE1AcxO9Iap+Vd2ojc8Djw80SXc9hhG2wVcpuJjVJi6Zy6oMOIb5Di64UI7lo6YmyRmBULrNvYtTa5zDC0Sug7TSTq2PMPZBcIwScwT4Vmdx307bAsSRt+wFGvMiD61DS8keEQyFJpUfjeIfabDOD/aRI3MWzCux4/JBShpFauhJuS8EjwkcYh3R1tAL1XP/YdFu/0dtnzgxd4VU5VgR8Mhh96/p+I29a5dlvsXxMVT3Rw1Ps/+9iJGmNEoyZMTXnzJAS9n8T3FaPX2qGhwb/hQfWmPL7LkYyAlwBRjvSPWtGJI9BcOCc1s0z0WwQ0wszz/2wN34XI035lmAcjDvvtOTNwJxmc5+M7iYXfwz+4z+f4AnjyxiT4Wg4AFd9Pbd6k29EK3zUysOynqKC4Iv/CqXrCQTKlzEmw1GzfwhkTLr0rTn5f529BIsHFpzVr5Ztoh1vsu/8MU/JQzW0r7+NkQzHvgjj00W2hoU2fdiSZipFyynpBaDnIOlN9q2/ifYcZNg5kTxGkPRZ7ZQ61qPSFb6ZuQouzMl+vY4Rvt7V10FhBv8hnIxbNIBlrq0O7cHXK49919Sohads0ctqT/603INdxf9/AVM0ixYwCugKAAAAAElFTkSuQmCC") left center no-repeat;
```

Comment:
This was done a long time ago with the intent of simply editing the CSS file, but I've gone a bit further since then by modifying a few template, images... So, I'm going to refactor this to use a real SVG instead of base64.